libs/cart/testing/src/factories/cart-item/cart-item.factory.ts

```diff
@@ -14,10 +14,19 @@ export class DaffMockCartItem implements DaffCartItem {
 	image = <DaffProductImage>new DaffProductImageFactory().create();
   sku = 'sku';
   name = 'Product Name';
-  qty = faker.random.number({min:1, max:100});
+  qty = faker.random.number({min: 1, max: 100});
   price = faker.random.number({min: 1, max: 1500});
   row_total = this.qty * this.price
 	in_stock = true;
+  _numberOfDiscounts = faker.random.number({min: 1, max: 5});
```

Comment:
I think this should be `min: 0, max: 2` just to be a little more realistic. I don't think more than 2 discounts will happen too often in production, and you'll want to see the zero case too

Reply:
good point, changed